core/src/main/scala/org/bykn/bosatsu/library/Command.scala

```diff
@@ -197,6 +197,22 @@ object Command {
           )
         }
 
+    val allLibs: Opts[F[(P, SortedMap[Name, (LibConfig, P)])]] =
```

Comment:
just change this to a function `def readAllLibs(gitRoot: P): F[SortedMap[Name, (LibConfig, P)]] = ` and not bake it in with the opts.